src/features/settings/settingsSlice.ts

```diff
@@ -112,8 +112,7 @@ export const getTdexdConnectUrl = createAsyncThunk<string, string, { state: Root
   'settings/getTdexdConnectUrl',
   async (password, thunkAPI) => {
     try {
-      if (!(window as any).TDEX_DAEMON_URL) throw new Error('TDEX_DAEMON_URL is missing');
-      const res = await fetch(`${(window as any).TDEX_DAEMON_URL}/tdexdconnect`, {
+      const res = await fetch(`${window.location.host}/tdexdconnect`, {
```

Comment:
we should do this only if `__IS_PACKAGED__` is true?

Reply:
getTdexdConnectUrl is only used in this context so I didn't add a check inside the function

Reply:
Check added